src/components/App/App.js

```diff
@@ -29,7 +30,7 @@ class App extends Component {
   render() {
     return (
       <main>
-        <h3>Photo Map App</h3>
+        <Header/>
```

Comment:
The `<Header />` should probably be outside the `<main>` element, semantically. You'll need a wrapper `<div>` as well.